app/views/users/_profile_tabs.html.erb

```diff
@@ -0,0 +1,13 @@
+      <ul class="nav nav-tabs" id="myTab">
+        <li class="active"><a href="#contributions"><i class="glyphicon glyphicon-star"></i> Contributions</a></li>
+        <li><a href="#profile"><i class="glyphicon glyphicon-info-sign"></i> Profile</a></li>
+        <% if current_user && (current_user.admin? || @user == current_user) %>
```

Comment:
Good idea! It took a bit (@current_user @user were both empty User class objects?!) to figure out the ability logic that would work.  Here it is for review!